src/main/java/ch/wisv/areafiftylan/security/JsonLoginFilter.java

```diff
@@ -94,4 +126,12 @@ protected String obtainPassword(HttpServletRequest request) {
     public AuthenticationManager getAuthenticationManager() {
         return this.authenticationManager;
     }
+
+    private String getClientIP(HttpServletRequest request) {
+        String xfHeader = request.getHeader("X-Forwarded-For");
```

Comment:
Does this need a readme section for deploying? Ensure that this header is present when forwarding in nginx? What if we are not deploying behind nginx?